src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -1059,4 +1057,54 @@ private void sendEvent(EventType eventType, EventPayload payload, Object caseId)
         .with("transactionId", transactionId)
         .debug("{} event published", eventType);
   }
+
+  private CaseDTO getLatestCaseById(UUID caseId, Boolean getCaseEvents) throws CTPException {
```

Comment:
I would prefer if this method was situated next to the other method it was based on (I think you renamed it to getLatestCaseByUPRN) . They are quite similar, so it would be easier to see ways to common up the code if possible. 

Also, as a point of style it is somewhat inconsistent to suffix this method "Id" (ie camel case) but the other method "UPRN" (all caps).  I always prefer camel case in all cases. It also has some advantages with eclipse shortcuts for finding classes, when you apply this to class names.

Reply:
yes - blame Phil : )

methods now renamed/moved as required